packages/frontend/src/components/FolderList/Folders.tsx

```diff
@@ -61,10 +58,9 @@ export const Folders = ({ isEditing }: FoldersProps) => {
     } catch (error) {
       setError(true);
     }
-    resetState();
   };
 
-  const editList = async (component: Component | null, title: string) => {
+  const editList = async (component: Node | null, title: string) => {
```

Comment:
What's the difference between `Component` and `Node` type here and packages/frontend/src/components/FolderList/Folders.tsx line 13?

Reply:
fixed